src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -191,15 +203,16 @@ void initializeReference() {
      */
     void initializeReads() {
         if (! readArguments.getReadFiles().isEmpty()) {
+            // Prefetcher is useful for cloud files because of the latencies involved.
+
```

Comment:
Stray comment left over from before the rebase should be deleted or moved to an appropriate place.